models/mapper.go

```diff
@@ -49,6 +53,14 @@ func RectifyKeywords(keywords []string, keywordsLimit int) []string {
 	return rectifiedKeywords[:keywordsLimit]
 }
 
+func ValidateTopics(topics []string) []string {
+	if topics == nil {
```

Comment:
The `topics` array can either be a nil slice or an empty slice and currently, this is only checking if the array is nil. I believe this validation is also applicable when the array is empty as well